packages/kbn-test/src/prettify_command_line.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { execSync } from 'child_process';
+import * as path from 'path';
+
+const kibanaRoot = execSync('git rev-parse --show-toplevel').toString().trim() || process.cwd();
+
+export function prettifyCommandLine(args: string[]) {
+  let [executable, ...rest] = args;
```

Comment:
Super duper nit, but why `let` over `const` here?

Reply:
Oh, I see the reassignment below, again this is a nit, but I'd just define a new arg, personally.

Reply:
Sorry, this came in after the bell rang :)

Reply:
Yeah no worries at all mate 👍🏾